MvvmCross_All.sln

```diff
@@ -407,13 +407,27 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "MvvmCross.Plugins.Visibilit
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "MvvmCross.Plugins.WebBrowser.Uwp", "MvvmCross-Plugins\WebBrowser\MvvmCross.Plugins.WebBrowser.Uwp\MvvmCross.Plugins.WebBrowser.Uwp.csproj", "{08E19EF4-8C86-4F6E-B78A-063983884434}"
 EndProject
-Project("{9344BDBB-3E7F-41FC-A0DD-8665D75EE146}") = "MvvmCross.Platform.Wpf", "MvvmCross\Platform\Wpf\MvvmCross.Platform.Wpf.csproj", "{BDD29D7B-293A-4A4A-8DC8-F18285CBFD72}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "MvvmCross.Platform.Wpf", "MvvmCross\Platform\Wpf\MvvmCross.Platform.Wpf.csproj", "{3688D6FC-E7F4-485F-8B27-96C0A8C7EAED}"
```

Comment:
I'm not sure, I only added the test projects to the main solution, I'll roll this id change back - nice catch!